### PR TITLE
Bump ai.onnx opset to 22

### PR DESCRIPTION
**Pull request by cjvolzka**
### Description
* Bump ai.onnx opset to 22

### Motivation and Context
* Bump opset to 22 for future 1.17.0 release now that 1.16 branch has been cut.

### CODE REVIEW

File: onnx/defs/operator_sets.h
Comment (justinchuby) [inline onnx/defs/operator_sets.h:1194]: Curious what this does?

Comment (cjvolzka) [reply onnx/defs/operator_sets.h:1194]: I'm not sure. I was following https://github.com/onnx/onnx/blob/main/docs/OnnxReleases.md#after-pypi-release "Bump opset version for ai.onnx" section.

I realized that those steps could be done after the release branch is cut instead of waiting for the release to go public. However it looks like it causes the Linux CIs report a failure when it detects noting yet uses the `fn` inside of the method.

@liqunfu is this something that should still be done preemptively or should this wait until there's actually a change related to the next opset?

Comment (jcwchen) [reply onnx/defs/operator_sets.h:1194]: > Curious what this does?

I can provide the history context here: without this previously ONNX developers might overlook creating new opset version here when adding new operators and there were a lot of back and forth communication for developers to change their PRs. Thus we think we can let the release manager always create the new opset here right after the release and in the future other developers won't need to worry about it.

**Review by justinchuby — APPROVED** [review]

**Comment by justinchuby** [comment]
@gramalingam

**Pull request merged**
